package.json

```diff
@@ -22,7 +22,8 @@
     "body-parser": "^1.18.3",
     "dotenv": "^6.2.0",
     "express": "^4.16.4",
-    "watson-developer-cloud": "^3.18.1"
+    "fs":"~0.0.1-security",
+    "watson-developer-cloud": "^3.13.1"
```

Comment:
You need to switch to use `ibm-watson:4.0.2` because the new features for Watson Assistant will only be available in the new SDK not the old one.